region/brinstar/blue.json

```diff
@@ -1223,16 +1179,13 @@
                   "notable": false,
                   "requires": [
                     "Morph",
-                      {"or": [
-                        "canConsecutiveWalljump",
-                        "h_canFly"
-                      ]}
+                    {"or": [
+                      "canConsecutiveWalljump",
+                      "h_canFly"
+                    ]},
+                    {"obstaclesCleared": ["B"]}
```

Comment:
This scenario already has "A" cleared so I'm wondering if `obstaclesCleared: A` shows intent better.  In the fantasy world of crumble broken and bomb block not, you could spark up for instance.
And there may be room for an ice clip strat.
GMode would fit the need for `h_canDestroyBombWalls`

Doesn't affect things either way as things are 🤷